gateway/tests/telemetry/metrics/request.rs

```diff
@@ -311,13 +311,12 @@ fn connected_clients() {
                 r#"
                 SELECT Value, Attributes
                 FROM otel_metrics_sum
-                WHERE ServiceName = ? AND StartTimeUnix >= ?
```

Comment:
For some reason the new otel crates are faster here than our start_time_unix. This will still fetch the latest for the given service name, the item is there and the test correct.